tests/models/test_simplemodel.py

```diff
@@ -39,3 +42,21 @@ def test_simplemodel(testdata_3days_path):
             track_b,
             round_lon_lats=True)
     assert rarest_measurement_pair
+
+    # test MeasurementPairClassifier
```

Comment:
misschien voor de volledigheid hierboven ook ergens een comment met 'test general utility functions' of zoiets?

Reply:
Ik zou het overigens ook geen gek idee vinden om elke functie een aparte test te geven, want wellicht dat we die functies uit simplemodel.py willen halen en in een aparte utils.py of zo willen zetten?

Reply:
Heb er losse functies van gemaakt!